src/test/resources/hiddenType/expected/com/example/Simple.java

```diff
@@ -0,0 +1,8 @@
+package com.example;
+import com.github.javaparser.ast.body.MethodDeclaration;
+
+public class Simple {
+    private boolean isVoidType (MethodDeclaration method) {
+        return method.getType().isVoidType();
+    }
+}
```

Comment:
Professor,

> The "expected" here needs to include:
> 
> the MethodDeclaration class

For `MethodDeclaration`, are we expecting an empty class with nothing inside or a class with a constructor method that throws an error?

> If possible, we'd really prefer to get the name of the class (i.e., Type) for the second item in the list correct. Can the JavaParser SymbolSolver resolve that?

I will try it and let you know.

Reply:
> For MethodDeclaration, are we expecting an empty class with nothing inside or a class with a constructor method that throws an error?

Neither: it needs to include a `getType` method (which will throw an error), because that method is used in the target program.